src/index.js

```diff
@@ -4,18 +4,20 @@ import './index.css'
 import App from './App'
 import { makeServer } from './server'
 import { BrowserRouter } from 'react-router-dom'
-import { ProductFilterProvider } from './context/product-context'
+import { ProductFilterProvider, WishlistProvider } from './context/'
 
 // Call make Server
 makeServer()
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
-      <ProductFilterProvider>
-        <App />
-      </ProductFilterProvider>
-    </BrowserRouter>
+      <WishlistProvider>
+        <ProductFilterProvider>
+          <App />
+        </ProductFilterProvider>{' '}
+      </WishlistProvider>{' '}
+    </BrowserRouter>{' '}
```

Comment:
You can remove curly braces. This is not needed.

Reply:
Done